web/src/styles/questions/question-one-styles.css

```diff
@@ -0,0 +1,73 @@
+.sequential-intro-1 {
+    position:fixed;
+    z-index: 1;
+    animation: disappear 0s forwards;
+    animation-delay: 7s;
+}
+
+@keyframes disappear {
```

Comment:
@alexanderlim9 Overall good use of CSS but I'm wondering if we can just use a library for these to make it smoother (https://www.npmjs.com/package/react-typist#cursor). If not, I think some of the transitions could be a bit shorter